### PR TITLE
Things done

### PR DESCRIPTION
**Pull request by harikt**
Base filter added to the docs

Multiple rules can be added to a field.

Form now has Iterator .

### CODE REVIEW

**Comment by harikt** [comment]
You were ok with `IteratorAggregator` for Form ? I am seeing things different in form branch.

**Comment by pmjones** [comment]
It's on `Collection` but I need to add it on `Fieldset` (which is the parent of `Form`).

**Comment by harikt** [comment]
ok.

File: src/Aura/Input/Filter.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Filter implements FilterInterface
      */
     public function setRule($field, $message, \Closure $closure)
     {
-        $this->rules[$field] = [$message, $closure];
+        $this->rules[$field][] = [$message, $closure];
```
Comment (harikt) [inline src/Aura/Input/Filter.php:39]: Addressing #31

**Pull request merged**